src/core/builtin_functions.cc

```diff
@@ -721,6 +721,25 @@ Value builtin_parent_module(Arguments arguments, const Location& loc)
   return {UserModule::stack_element(s - 1 - n)};
 }
 
+Value builtin_evalstring(const std::shared_ptr<const Context>& context, const FunctionCall * func_call)
```

Comment:
Please keep that separate, it's not clear if that's going to be merged, so we don't want to combine it with the Python changes.